Enterprise_Dbcache_WpdbInjection_Cluster.php

```diff
@@ -575,7 +577,7 @@ function query( $query ) {
 			if ( preg_match( '/^\s*SELECT\s+SQL_CALC_FOUND_ROWS\s/i', $query ) ) {
 				if ( false === strpos( $query, "NO_SELECT_FOUND_ROWS" ) ) {
 					$this->wpdb_mixin->timer_start();
-					$this->wpdb_mixin->_last_found_rows_result = mysqli_query( $this->wpdb_mixin->dbh, "SELECT FOUND_ROWS()" );
+					$this->wpdb_mixin->_last_found_rows_result = mysqli_num_rows( $this->wpdb_mixin->dbh, "SELECT FOUND_ROWS()" );
```

Comment:
This fixes pagination not working as this is the only way to properly fetch `SELECT FOUND ROWS()`

Reply:
Actually, this worked but was still incorrect. I pushed a new update to solve this.